lib/reverse_sentence.rb

```diff
@@ -1,6 +1,38 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2), where n is the length of my_sentence
```

Comment:
Incorrect. The outer loop is O(n), all the substring reversals together all add up to another O(n), and the final string reverse is another O(n). All together, that's still O(n).

Also, just to be pedantic, n is the length of your sentence *in characters*.